gpu4pyscf/scf/_response_functions.py

```diff
@@ -32,6 +32,9 @@ def _gen_rhf_response(mf, mo_coeff=None, mo_occ=None,
     if mo_occ is None: mo_occ = mf.mo_occ
     mol = mf.mol
     if isinstance(mf, hf.KohnShamDFT):
+        grids = mf.cphf_grids
+        if grids.coords is None:
```

Comment:
This will affect all linear response methods and many features, not just the CPHF solver. Consider to pass this as an optional argument.

Reply:
Done